src/pages/Home/Home.jsx

```diff
@@ -1,31 +1,57 @@
-import React from 'react';
-
-// import { Footer, ErrorBoundary, Header, Search } from '../../Components';
+import React, { Suspense, lazy } from 'react';
+import { Modal } from 'antd';
+import { Actions } from '../../context/ModalContext';
+import { useAddDeleteModal } from '../../hooks/useAddDeleteModal';
 import {
   Footer,
   Header,
   Search,
   ErrorBoundary,
   Layout,
-  TabsMovies
+  AddEditContent
 } from '../../components';
-// import styles from './Home.modules.scss';
+
+const TabsMovies = lazy(() =>
+  import('../../components').then((module) => ({
+    default: module.TabsMovies
+  }))
+);
 
 /**
  * Renders the main Home of the APP
  * @returns
  */
 
-export const Home = () => (
-  <div>
-    <Header>
-      <ErrorBoundary>
-        <Search details={[]} />
-      </ErrorBoundary>
-    </Header>
-    <Layout>
-      <TabsMovies />
-    </Layout>
-    <Footer />
-  </div>
-);
+export const Home = () => {
+  const { state, updateModalType } = useAddDeleteModal();
+  const { openModal } = state;
+  const handleCancel = () => {
+    updateModalType(Actions.CLOSE_MODAL);
+  };
+
+  return (
+    <div>
+      <Header>
+        <ErrorBoundary>
+          <Search details={[]} />
```

Comment:
Better to make details as an empty array inside the `Search` component with using `propTypes`:

```
Search.defaultProps = {
  details: [ ]
};
```